src/main/java/org/isf/opd/service/OpdIoOperationRepositoryCustom.java

```diff
@@ -25,10 +25,11 @@
 
 import java.util.GregorianCalendar;
 import java.util.List;
+import java.sql.Date;
 
 public interface OpdIoOperationRepositoryCustom {
 
-	List<Opd> findAllOpdWhereParams(String diseaseTypeCode, String diseaseCode, GregorianCalendar dateFrom,
-									GregorianCalendar dateTo, int ageFrom, int ageTo, char sex, char newPatient);
+	List<Opd> findAllOpdWhereParams(String diseaseTypeCode, String diseaseCode, Date dateFrom,
+									Date dateTo, int ageFrom, int ageTo, char sex, char newPatient);
```

Comment:
correct, `dateFrom` is (in the other class) compared with `o.visitDate` so `java.sql.Date == DATE`